src/scopes/activityReport/myReports.js

```diff
@@ -30,6 +32,11 @@ export function myReportsScopes(userId, roles, exclude) {
     WHERE "ActivityReportApprovers"."userId" = '${userId}'`;
   }
 
+  if (!reportSql.trim()) {
```

Comment:
```suggestion
  if (!reportSql.trim) {
```

```number.trim()``` throws an error, ```number.trim``` returns undefined (I think)

Reply:
You're right that calling .trim() on a non-string value (like a number) would throw an error. However, reportSql is always a string in this context, so .trim() is safe. That said, since reportSql is always initialized as an empty string (''), I can remove .trim() altogether for clarity.